tomls/omnibus-base-mainnet-andromeda/perps/ena.toml

```
@@ -5,7 +5,7 @@ defaultValue = "0xb7910ba7322db020416fcac28b48c01212fd9cc8fbcbaf7d30477ed8605f6b
 defaultValue = "700"
 
 [setting.perpsEnaSkewScale]
-defaultValue = "34000000"
+defaultValue = "25500000"
```

Here and for all other similar places: for bigger numbers for readability it is worth using different format
```suggestion
defaultValue = "<%= String(25_500_000) %>"
```

---

this is pretty nice tbh, yes

---

will do it on all after we merge this. for the sake of no bad inputs.